File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.swift
import Foundation

func solution(_ numbers:[Int]) -> Int {
    var newArr: [Int] = Array(repeating: 0, count: 10)
    var ans = 0
    
    for i in numbers.sorted() {
        newArr[i] = i
    }
    
    for i in newArr.enumerated() {
        if i.element == 0 {
            print(i.offset)
            ans += i.offset
        }
    }
    return ans
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
//         int answer = 0;
//         boolean[] isExisted = new boolean[10];
//         for(int i=0; i<numbers.length; i++) {
//             isExisted[numbers[i]] = true;
//         }
        
//         for(int i=0; i<isExisted.length; i++) {
//             if(!isExisted[i]) answer += i;
//         }
//         return answer;
        int totalSum = 45;
        int sum = 0;
        for(int i: numbers) {
            sum += i;
        }
        return totalSum - sum;
    }
}